frontend/src/app/task-area/task-area.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+import { TaskLoaderService, TaskInterface } from '../task-loader.service';
+@Component({
+  selector: 'app-task-area',
+  templateUrl: './task-area.component.html',
+  styleUrls: ['./task-area.component.scss'],
+})
+
+export class TaskAreaComponent implements OnInit {
+  displayedTask: TaskInterface[] = [];
```

Comment:
better displayedTasks <- everybody will understand that it is an array/list (not a single item)

Reply:
fixed